### PR TITLE
Fixes disabling of selectable / multi select / actions

### LINKED ISSUE #1534: selectable repeater does not respect disabled state
use jsbin for testing. currently fuelux.min.js points to local. http://jsbin.com/kucifivufo/1/edit?html,js,output

### PR DESCRIPTION
**Pull request by BenjaminNeilDavis**
Fixes #1534

### CODE REVIEW

File: js/repeater-list.js [hunk 1]
```diff
@@ -594,6 +594,7 @@
 		if (sortable) {
 			$both.addClass('sortable');
 			$div.on('click.fu.repeaterList', function () {
+				if(self.isDisabled) return;
```
Comment (kevinparkerson) [inline js/repeater-list.js]: this is technically correct, but as per our style guide we are supposed to write it as follows:
`if (self.isDisabled) { return; }`

File: js/repeater-list.js [hunk 2]
```diff
@@ -655,6 +656,7 @@
 			$row.data('item_data', rows[index]);
 
 			$row.on('click.fu.repeaterList', function () {
+				if(self.isDisabled) return;
```
Comment (kevinparkerson) [inline js/repeater-list.js]: this is technically correct, but as per our style guide we are supposed to write it as follows:
if (self.isDisabled) { return; }

**Comment by kevinparkerson** [comment]
Looks good outside minor style guide compliance changes! :+1:

**Comment by futuremint** [comment]
@BenjaminNeilDavis looks like there is an issue with the unit tests: https://travis-ci.org/ExactTarget/fuelux/builds/81997536#L519 Can you please fix?

**Comment by cormacmccarthy** [comment]
https://github.com/cormacmccarthy/fuelux/tree/BenjaminNeilDavis-issue-1534-diable-repeater

I've updated the syntax of your async tests @BenjaminNeilDavis. However, there is still some sort of race condition keeping your tests from completing consistently and therefore when run in the context of all tests it fails (however, when run alone, it succeeds).

You can check the tests locally by running `grunt servefast` and then go http://localhost:8000/test/

**Pull request closed without merging**